src/ert/scheduler/lsf_driver.py

```diff
@@ -105,7 +128,7 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, Dict[str, Dict[str, str]]]:
                 )
                 continue
             data[job_id] = {"job_state": job_state}
```

Comment:
Would it simplify the code if we returned the correct Job classes here, and not the string `job_state`?